Source/WTF/wtf/PlatformUse.h

```diff
@@ -376,3 +376,10 @@
     || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MAX_ALLOWED >= 90000)
 #define USE_VARIABLE_OPTICAL_SIZING 1
 #endif
+
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED < 150000) \
+    || (PLATFORM(IOS_FAMILY) && __IPHONE_OS_VERSION_MAX_ALLOWED < 180000) \
```

Comment:
If these `*_MAX_ALLOWED` macros are undefined, they'll be treated as zero - is it possible for `PLATFORM(IOS_FAMILY)` to be defined but `__IPHONE_OS_VERSION_MAX_ALLOWED` to be undefined? Maybe in the case of a non-iPhone IOS_FAMILY OS?

Elsewhere we use `*_MAX_ALLOWED` as the left hand side of a > or >= expression which helps avoid the undefined-macro-is-zero situation.

Reply:
The `_MAX_ALLOWED` macros are all guarded by `PLATFORM` macros, so I don't think there is a problem. Historically, this is the pattern we've used for workarounds that are not necessary in perpetuity, and we have an existing example of this pattern in `PlatformHave.h` right now.